src/components/index.ts

```diff
@@ -0,0 +1,11 @@
+export { default as Button } from './button';
```

Comment:
Is this needed if the component's index file includes `export { default as Button } from './Button';`? Or can this file just be a conglomeration of `export * from './whatever';` statements?

Reply:
Actually I misspoke before. This is correct as is, unless of course we want to refactor existing imports. 

Within src/components/button
`export { default } from './Button';`

So that consumers like src/components/primary-button or parent apps will do (what they already do)
`import Button from '../button';`
`import Button from 'buie/es/components/button';`

If instead inside src/components/button you do `export { default as Button } from './Button';` then consumers will have to do `import { Button } from '../button';`

Now in src/components/index.ts nothing can be default, so default from "button" folder is exported as named. Going up, src/index.ts will just re-export that named export using *. So the parent consumer will then do `import { Button } from 'buie';`